### DISCUSSION TITLE
Error: No loader is  configured

### DISCUSSION THREAD

**Discussion by ikarus-akhil** [remix-run/remix#4429] [Q&A] [answered]
I am trying to put 3D Models on my website and using model-viewer. But when I run the build command I get the error that `No loader is configured for ".glb" files` and ` ERROR: No loader is configured for ".hdr" files`. How can I resolve this error?

**Comment by sergiodxa** [comment, marked as answer]
The loaders are not configurable right now, I opened a discussion to propose an idea to let you import any file as a hashed URL or plain string with the content https://github.com/remix-run/remix/discussions/4432

**Reply by machour** [reply]
Related PR from OP: https://github.com/remix-run/remix/pull/4457
Going to mark this an answered
